src/log-store/Cargo.toml

```diff
@@ -21,10 +21,14 @@ common-macro.workspace = true
 common-meta.workspace = true
 common-runtime.workspace = true
 common-telemetry.workspace = true
+dashmap = "5.4"
 futures-util.workspace = true
 futures.workspace = true
 protobuf = { version = "2", features = ["bytes"] }
 raft-engine.workspace = true
+rskafka = "0.5"
```

Comment:
Looks like the `rskafka` is used in several crates, I think it can be moved to the project `cargo.toml` and use `rskafka.workspace = true` instead.

Reply:
Moved `rskafka` to workspace cargo toml.